### ISSUE TITLE
[Bug]: can't open the app - it says 'REPAIR'

### ISSUE THREAD

**Issue by scb59** [jasp-stats/jasp-issues#2802]
### JASP Version

0.18.3

### JASP Module

Other

### What OS are you seeing the problem on?

Windows 11

### Bug Description

I downloaded JASP on my HP laptop and every time I try to open it, it says it needs to 'repair' . I saw on a 2021 blog post that this was an issue but no answer given. What can I do?

### Expected Behaviour

The app should open straight away. 

### Steps to Reproduce

1. I unstalled the app
2. I reinstalled the app
3. same issue
...

### Final Checklist

- [ ] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
You could
- show us the screenshot of the error message. It will certainly say more than "repair"
- if you have an extra antivirus-program running, deactivate it temporarily or even switch to windows defender (included in win11)
- deinstall JASP and reintall it after making sure that win11 is fully up to date
- install via the windows store (the black button) instead via the direct download - both install methods are available via jasp-stats.org/download

**Comment by boutinb** [comment]
If it says repair, this means that JASP is not fully uninstalled: Windows still thinks (in its registry) that JASP is installed.
What I would like also to know: what happens if you click on 'Repair' ?

**Comment by tomtomme** [comment]
@scb59 
any news for us?

**Comment by scb59** [comment]
No, I still can't download it. and the online version is now telling me
that there is an issue with JASP and you are looking into it. The only
thing I can do is use it as a chrome extension.

Thanks

Sophie


Mobile: +44(0)7515119707

Website: www.mindfullyme.co.uk

**Comment by boutinb** [comment]
You can also try the Microsoft store installation. There is also a pre-installed version: this is just a zip file you can unzip afterwards: https://jasp-stats.org/download/.

**Comment by tomtomme** [comment]
@scb59 
you could also try to re-test the installation with the new JASP 0.19.2!
Thx
